grtext/grfont.cpp

```diff
@@ -655,7 +654,7 @@ bool grfont_SetTemplate(const char *pathname, const tFontTemplate *ft) {
 
   //	Write widths now if necessary.(FT_PROPORTIONAL)
   if (fnt.flags & FT_PROPORTIONAL) {
-    for (int i = 0; i < num_char; i++)
+    for (i = 0; i < num_char; i++)
```

Comment:
where is that `i` declared? It would be better to keep it a loop variable

Reply:
There `i` usage out of scope in `grfont_SetTemplate()` after loop. This is only cheap and quick way to silence "overshadowed local variable" warning.